static/js/src/advantage/users/components/TableView/components.tsx

```diff
@@ -0,0 +1,192 @@
+import React from "react";
+import { format } from "date-fns";
+import CSS from "csstype";
```

Comment:
I don't see this dep in the package.json. It's best to be explicit about our dependencies, can you add it there plz.

Reply:
Will do!